lib/parse-all.js

```diff
@@ -26,9 +26,15 @@ function parseAll(config) {
 					return { name: name, error: err };
 				});
 		})
-	);
+	).then(result => {
+		const errors = result.filter(error => error);
```

Comment:
Result is an array of the errors thrown by a parser. If a parser does not throw an error, then the item in the array is `undefined`. So say that only parser `jsmin` throws an error, the array would look like this:
```
[
undefined,
undefined,
undefined,
{error object},
undefined,
...
]
```

This filter results in an array with only errors.

The naming of `result` might not be quite optimal.